pkg/serverless/random/rand.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016 Datadog, Inc.
 
-package invocationlifecycle
+package random
```

Comment:
I think we can get rid of this package

Reply:
Do you  mind explaining why? I was under the impression that we needed to have a the random variable for thread safety. This was located inside another package, but to access it elsewhere, I moved it to its own package.

Reply:
It looks like rand package has a built-in method to construct random Uint64 : https://pkg.go.dev/math/rand#Rand.Uint64
Maybe it's worth asking the tracing team to be sure it's safe to use it, if yes, we could get rid of this package

Reply:
Spoke about this privately. To avoid concurrency issues we will keep the package.